rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -62,6 +63,11 @@
       "type": "string",
       "minLength": 4
     },
+    "second_call_number": {
```

Comment:
Is it going to be mandatory to have a  `first_call_number` in order to have the `second_call_number`?

Reply:
I added a validation to check that